lambda/locals.tf

```diff
@@ -1,44 +1,46 @@
-  sign_cookies_arn                      = var.apply_resource == true && var.lambda_sign_cookies == true && length(aws_lambda_function.sign_cookies_lambda_function) > 0 ? aws_lambda_function.sign_cookies_lambda_function.*.arn[0] : ""
+  workspace                                 = lower(terraform.workspace)
+  environment                               = local.workspace == "default" ? "mgmt" : local.workspace
+  count_av_yara                             = var.apply_resource == true && var.lambda_yara_av == true ? 1 : 0
+  count_checksum                            = var.apply_resource == true && var.lambda_checksum == true ? 1 : 0
+  count_log_data                            = var.apply_resource == true && var.lambda_log_data == true ? 1 : 0
+  count_api_update                          = var.apply_resource && var.lambda_api_update == true ? 1 : 0
+  count_file_format                         = var.apply_resource && var.lambda_file_format == true ? 1 : 0
+  count_log_data_mgmt                       = var.apply_resource == true && var.lambda_log_data == true && local.environment == "mgmt" ? 1 : 0
+  count_download_files                      = var.apply_resource == true && var.lambda_download_files == true ? 1 : 0
+  count_notifications                       = var.apply_resource == true && var.lambda_ecr_scan_notifications == true ? 1 : 0
+  count_ecr_scan                            = var.apply_resource == true && var.lambda_ecr_scan == true ? 1 : 0
+  count_sign_cookies                        = var.apply_resource == true && var.lambda_sign_cookies == true ? 1 : 0
+  count_efs                                 = var.apply_resource == true && var.use_efs ? 1 : 0
+  count_create_db_users                     = var.apply_resource == true && var.lambda_create_db_users ? 1 : 0
+  count_create_keycloak_db_user             = var.apply_resource == true && var.lambda_create_keycloak_db_users ? 1 : 0
+  count_create_keycloak_db_user_new         = var.apply_resource == true && var.lambda_create_keycloak_db_users_new ? 1 : 0
+  count_export_api_authoriser               = var.apply_resource == true && var.lambda_export_authoriser == true ? 1 : 0
+  count_service_unavailable                 = var.apply_resource == true && var.lambda_service_unavailable == true ? 1 : 0
+  api_update_function_name                  = "${var.project}-api-update-${local.environment}"
+  checksum_function_name                    = "${var.project}-checksum-${local.environment}"
+  create_db_users_function_name             = "${var.project}-${var.lambda_name}-${local.environment}"
+  create_keycloak_db_user_function_name     = "${var.project}-create-keycloak-db-user-${local.environment}"
+  create_keycloak_db_user_new_function_name = "${var.project}-create-keycloak-db-user-new-${local.environment}"
```

Comment:
Maybe the `new` can be added to the end of the local name to make it more obvious and consistent?